src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
```

Comment:
There's an unnecessary empty line at the start of the class implementation. Please remove it to adhere to the coding standards.